sdk/samples/iot/provisioning/src/paho_iot_provisioning_example.c

```diff
@@ -111,7 +111,7 @@ int main()
           rc = az_iot_provisioning_client_get_client_id(
               &provisioning_client, mqtt_client_id, sizeof(mqtt_client_id), NULL)))
   {
-    printf("Failed to get MQTT clientId, az_result return code %04x\n", rc);
+    (void)printf("Failed to get MQTT client id: az_result return code %04x .\n", rc);
```

Comment:
Nit: Will %04x print something like 0xDEADC0DE or do we need to add the 0x like `0x%04x`? 
I remember a nightmare debugging session a long time ago where my mistake was not adding the "h" descriptor in an assembly DD instruction. Took me a week to find the bug so I am very careful now.

Reply:
i think you're right.  I need to add 0x in front of %
Thanks